### PR TITLE
Adds docker

### PR DESCRIPTION
**Pull request by tarekziade**
- [x] Makes it possible to use Docker
- [x] add a docker push in our CI
- [x] document how to use the docker image with a config file.

#### Pre-Review Checklist
- [ ] Covered the changes with automated tests
- [ ] Tested the changes locally
- [ ] Added a label for each target release version (example: `v7.13.2`, `v7.14.0`, `v8.0.0`)
- [ ] Considered corresponding documentation changes
- [ ] Contributed any configuration settings changes to the configuration reference

#### Changes Requiring Extra Attention

- [ ] Security-related changes (encryption, TLS, SSRF, etc)
- [ ] New external service dependencies added.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -56,3 +56,9 @@ ftest: bin/pytest bin/elastic-ingest
 
 run: install
 	bin/elastic-ingest --debug
+
+docker:
```
Comment (timgrein) [inline Makefile:60]:
```suggestion
docker-build:
```

Maybe call this `docker-build` to keep it consistent with `docker-run`?

File: docs/docker.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# How to run the service in docker
+
+Make sure you have the latest `docker` command, then build the image:
+
+    make docker
+
+
+To use it, tweak your configuration file and run:
```
Comment (timgrein) [inline docs/docker.md:8]:
```suggestion
To use it, tweak your [configuration file](./CONFIG.md) and run:
```

I suggest we add a link to the config file description here.

**Review by timgrein — APPROVED** [review]
Two small comments, but otherwise LGTM.

**Comment by tarekziade** [comment]
Try run in buildkite -- see https://buildkite.com/elastic/connectors-python-nightly/builds/298#01862954-48ed-491a-a4e2-b09b5b17cc56

**Pull request merged**